guide/designing-products/personal-finance.md

```diff
@@ -226,11 +231,12 @@ Many people don’t self-cusody their long-term savings. They rely on banks and
    layout = "float-left"
 %}
 
-Investing is about putting saved money to work hoping that it increases in value, from retirement accounts to stock investing and real estate ownership. This necessarily requires the involvement of third parties, which goes beyond the current focus of this guide.
+Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide (at least for now).
```

Comment:
```suggestion
Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide.
```

Reply:
It reads well, but I really don't think the Bitcoin Design Guide will cover investing.